packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -99,7 +99,7 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
 
   private _getIconSpan(): JSX.Element {
     return (
-      <div className={this._classNames.iconContainer}>
+      <div className={this._classNames.iconContainer} aria-hidden="true">
```

Comment:
nit: can avoid redundancy 
```suggestion
      <div className={this._classNames.iconContainer} aria-hidden>
```